src/main/webapp/app/pages/newsPage/ChangedAnnotatonListItem.tsx

```diff
@@ -55,6 +56,38 @@ export const ChangedAnnotationListItem = (props: {
       break;
   }
 
+  // find the index of the gene column
+  const geneColumnIndex = annotationColumnHeader.findIndex(
+    column => column.name === 'Gene'
+  );
+  // transform the gene input to a link
+  props.data.forEach(row => {
+    const geneInput = row.content[geneColumnIndex].content;
+    console.log('geneInput', geneInput);
+    if (typeof geneInput === 'string') {
+      const tokens = geneInput.split(',');
+      if (tokens.length > 1) {
+        const itemLinks = tokens.map((token, i) => {
+          token = token.trim();
+          if (i === tokens.length - 1) {
+            return <GenePageLink hugoSymbol={token} />;
+          }
+          return (
+            <>
+              <GenePageLink hugoSymbol={token} />
+              {', '}
+            </>
+          );
+        });
+        row.content[geneColumnIndex].content = itemLinks;
+      } else {
+        row.content[geneColumnIndex].content = (
+          <GenePageLink hugoSymbol={geneInput} />
+        );
+      }
+    }
```

Comment:
Let's move this logic into a util method so it can be shared with UpdatedTxImplListItem

Reply:
This logic can also be simplified if you use `<WithSeparator/>`